File: spring-rabbit-tutorials/async-demo/bar-service/src/main/java/com/example/barservice/CacheConfiguration.java
package com.example.barservice;

import com.hazelcast.config.Config;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Profile;

@Configuration
@EnableCaching
public class CacheConfiguration {

    @Bean
    @Profile("default")
    public Config config() {
        return new Config().setClusterName("local");
    }
}

File: spring-rabbit-tutorials/async-demo/bar-service-kotlin/src/main/kotlin/com/example/CacheConfiguration.kt
package com.example

import com.hazelcast.config.Config
import org.springframework.cache.annotation.EnableCaching
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Profile

@Configuration
@EnableCaching
class CacheConfiguration {

    @Bean
    @Profile("default")
    fun config(): Config {
        return Config().setClusterName("local")
    }
}